File: 001-100/010/summationOfPrimes.php
<?php
  // Check if a number is prime
  function isPrime($n){
    for ($i = 2; $i < $n; $i++) {
      if($n % $i == 0){ return False; }
    }
    return True;
  }
  // Get the summation of primes
  function summationOfPrimes($n){
    $sum = 0;
    for($i = 0; $i < $n; $i++){
      if(isPrime($i)){ $sum += $i; }
    }
    return $sum;
  }
  // Get user input
  $prompt = "Enter an integer n: ";
  $n = readline($prompt);
  $sum = summationOfPrimes($n);
  echo "\nSummation of primes below $n";
  echo "\n$sum\n"
?>

File: 001-100/010/summationOfPrimes.py
from math import sqrt, ceil

# Check if a number is prime
def isPrime(n):
    for i in range(2, ceil(sqrt(n))):
        if(int(n) % i == 0):
            return False
    return True
# Get the summation of primes
def summationOfPrimes(n):
    sum = 0
    for i in range(1, int(n)):
        if(isPrime(i)):
            sum += i
    return sum

if __name__ == "__main__":
    # Get user input
    n = input("Enter an integer n: ")
    # Print results
    print("Summation of primes below " + "{:,}".format(int(n)))
    print("{:,}".format(summationOfPrimes(n)))
